<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyramidGame</class>
 <widget class="QWidget" name="PyramidGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>550</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #0F0F0F;
}
QPushButton {
	color: rgb(185, 185, 185);
	border-radius: 10px;
	background-color: #232D3F;
}

QPushButton:hover {
	background-color: #008170;
}

QLabel {
	color: rgb(209, 209, 209);
	border-radius: 10px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="4,0,0,0,0,1,3">
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButton3">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="pushButton9">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton6">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLabel" name="player1Label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton7">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButton8">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="5" alignment="Qt::AlignRight|Qt::AlignVCenter">
    <widget class="QPushButton" name="restartButton">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource/resource.qrc">
       <normaloff>:/icons/redo.png</normaloff>:/icons/redo.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pushButton4">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton5">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton2">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="4" colspan="2">
    <widget class="QLabel" name="player2Label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="vsLabel">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resource/resource.qrc">:/icons/vs.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignLeft|Qt::AlignVCenter">
    <widget class="QPushButton" name="homeButton">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource/resource.qrc">
       <normaloff>:/icons/home.png</normaloff>:/icons/home.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
  <include location="../resource/resource.qrc"/>
  <include location="../resource/resource.qrc"/>
  <include location="C:/Users/sydr3/.designer/resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
